### ISSUE TITLE
fail build

### ISSUE THREAD

**Issue by sl1pkn07** [papylhomme/diskmonitor#16]
```
[ 26%] Building CXX object settings/CMakeFiles/libsettings.dir/configdialog.cpp.o
In file included from /tmp/makepkg/diskmonitor/src/diskmonitor/settings/appearance.h:24:0,
                 from /tmp/makepkg/diskmonitor/src/diskmonitor/settings/configdialog.cpp:26:
/tmp/makepkg/diskmonitor/src/build/settings/ui_appearance.h:1:10: fatal error: klocalizedstring.h: No such file or directory
 #include <klocalizedstring.h>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [settings/CMakeFiles/libsettings.dir/build.make:79: settings/CMakeFiles/libsettings.dir/configdialog.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:719: settings/CMakeFiles/libsettings.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

```
└───╼  locate klocalizedstring.h
/usr/include/KF5/KI18n/klocalizedstring.h
└───╼  ls /usr/include/KF5/KI18n/klocalizedstring.h
-rw-r--r-- 1 root root 93516 mar  3 11:03 /usr/include/KF5/KI18n/klocalizedstring.h
└───╼  yaourt -Qi ki18n | grep Ver
Versión                   : 5.44.0-1

```

**Comment by papylhomme** [comment]
I just released a new version `0.3.2`, can you check again ? (problem was solved by #12 but never made it to a new release, my bad)

**Comment by sl1pkn07** [comment]
yep. build fine now

thanks!!
